tests/http/test_http_adapter.py

```diff
@@ -0,0 +1,23 @@
+import random
+
+import pytest
+
+from up42.http.config import ResilienceSettings
+from up42.http.http_adapter import create as create_adapter
+
+
+@pytest.mark.parametrize("include_post", [True, False])
+def test_should_create_adapter(include_post):
+    total_retries = 5
+    backoff_factor = 0.4
+    statuses = (random.randint(400, 600),)
```

Comment:
In the test we can use any value. The reason why I included only 429 in the actual configuration because the only 4xx exception we expect to retry is 429 due to rate limiting. 401 should be eliminated due to automatic token refresh. 403 can't be healed via retry in general. All other 4xx exception are meaningful errors we need to surface and handle with the corresponding domain exceptions.